File: Gherkin/Ast/DocString.cs
using System;
using System.Linq;

namespace Gherkin.Ast
{
	public class DocString : StepArgument, IHasLocation
	{
		public Location Location { get; private set; }
		public string ContentType { get; private set; }
		public string Content { get; private set; }
	}
}

File: java/src/main/java/gherkin/ast/DocString.java
package gherkin.ast;

public class DocString extends Node {
    private final String contentType;
    private final String content;

    public DocString(Location location, String contentType, String content) {
        super(location);
        this.contentType = contentType;
        this.content = content;
    }

    @Override
    public void accept(Visitor visitor) {
        visitor.visitDocString(this);
    }

    public String getContent() {
        return content;
    }
}
